evm/src/executor/backend/mod.rs

```diff
@@ -65,6 +65,9 @@ type ForkLookupIndex = usize;
 const DEFAULT_PERSISTENT_ACCOUNTS: [H160; 3] =
     [CHEATCODE_ADDRESS, DEFAULT_CREATE2_DEPLOYER, CALLER];
 
+const GLOBAL_FAILURE_SLOT: &str =
```

Comment:
needs docs that this is keccak("failed")

Reply:
done! https://github.com/foundry-rs/foundry/pull/5404/commits/a75f9fbf5d790b737240ba89bcdc15b30307d157

a note: this is actually only "failed" in bytes and padded—not `keccak("failed")`